addons/mail/static/src/core/common/thread_model.js

```diff
@@ -1136,6 +1136,7 @@ export class Thread extends Record {
                 { pinned: false }
             );
         }
+        e?.stopPropagation();
```

Comment:
Is it on purpose that you put it after the `return` above, potentially not doing the stop?

Reply:
Not on purpose, even though I noticed that the `return` statement does not get called when unpinning livechat chats.
My solution might be more of a workaround than an actual solution as I noticed that unpinning normal discuss chats works correctly even if the click event gets propagated and the function to open the thread gets called.

Reply:
Livechat from history have `isLocallyPinned` true but `is_pinned` false (there is no channel member).